docs/File-Formats.md

```diff
@@ -499,6 +499,9 @@ The mutation metadata file should contain the following fields:
 8. **data_filename**: your data file
 9. **gene_panel**: optional gene panel stable id
 10. **swissprot_identifier (optional)**: either `accession` or `name`, indicating the type of identifier in the `SWISSPROT` column
+11. **variant_classification_filter (optional)**: mutation types to be filtered out, separated by commas, using the allowed values specified below. By default, cBioPortal filters out the following mutation types: Silent, Intron, IGR, 3'UTR, 5'UTR, 3'Flank and 5'Flank, except the promoter mutations for the TERT gene. If no types want to be filtered out, leave the field empty.
```

Comment:
If you leave it empty, will no types be filtered out or will the default cBioPortal filtering apply? Please clarify.